apis/r/NEWS.md

```diff
@@ -1,4 +1,4 @@
-# tiledbsoma 1.5.1
+# tiledbsoma 1.5.2
```

Comment:
Maybe add a line here that the update was procedural to keep Py + R aligned?   Better than a gap?

Reply:
pushed